discord/app_commands/commands.py

```diff
@@ -1274,6 +1276,35 @@ async def on_error(self, interaction: Interaction, error: AppCommandError) -> No
 
         pass
 
+    def error(self, coro: ErrorFunc) -> ErrorFunc:
+        """A decorator that registers a coroutine as a local error handler.
+
+        The local error handler is called whenever an exception is raised in a child command.
+        The error handler must take 2 parameters, the interaction and the error.
+
+        The error passed will be derived from :exc:`AppCommandError`.
+
+        Parameters
+        -----------
+        coro: :ref:`coroutine <coroutine>`
+            The coroutine to register as the local error handler.
+
+        Raises
+        -------
+        TypeError
+            The coroutine passed is not actually a coroutine, or is an invalid coroutine.
+        """
+
+        if not inspect.iscoroutinefunction(coro):
+            raise TypeError('The error handler must be a coroutine.')
+
+        params = inspect.signature(coro).parameters
+        if len(params) != 2:
+            raise TypeError('The error handler must have 2 parameters.')
+
+        self.on_error = coro
```

Comment:
```suggestion
        self.on_error = coro  # type: ignore
```

Reply:
Oh that's what I was missing. Thank you. Is there some way that I could run the linter for myself before asking for review on a PR? I feel bad asking for review 20 times for tiny updates.

Reply:
CI is mainly `black` (`pip install -U black` and then `black discord examples` in the root of this project) and `pyright`. You can install `pyright` using `npm`. `npm install -g pyright@1.1.242` for the one used for CI and then use `pyright` in the root of the project.